docs/tutorials/authentication-github.rst

```diff
@@ -341,9 +341,9 @@ It is much more convenient than Basic Auth identifiers!
 Cache the token validation
 ''''''''''''''''''''''''''
 
-Using the following snippet you can cache the association between a token and the user id, in order to avoid requesting Github at each request.
+Using the following snippet you can cache the association between a token and the user id, in order to avoid requesting Github each time.
```

Comment:
I'd prefer to change `requesting Github` to `making a request to Github each time`.